api/app/Generators/ApplicationDocGenerator.php

```diff
@@ -48,6 +48,8 @@ public function generate(): self
             'generalQuestionResponses' => ['generalQuestion'],
         ]);
 
+        // pull data from application snapshot
+        // mirrors logic found in PoolCandidateCsvGenerator
```

Comment:
An attempt to make it so when changing it in one place, the other location is also considered